common/rabbitmq/templates/_helpers.tpl

```diff
@@ -87,3 +87,12 @@ rabbit://{{ default "" $envAll.Values.global.user_suffix | print $rabbitmq.users
                 values:
                 - reinstalling
 {{- end }}
+
+{{- define "sharedservices.labels" }}
+app.kubernetes.io/name: {{ .Chart.Name }}
+app.kubernetes.io/instance: {{ .Chart.Name }}-{{ .Release.Name }}
+app.kubernetes.io/version: {{ .Chart.Version }}
+app.kubernetes.io/component: RabbitMQ
+app.kubernetes.io/part-of: {{ .Release.Name }}
+app.kubernetes.io/managed-by: Helm
```

Comment:
Helm [sets this label](https://github.com/helm/helm/blob/14d0c13e9eefff5b4a1b511cf50643529692ec94/pkg/chartutil/create.go#L503) on its own. Please remove it to avoid patch/merge errors during upgrades. Helm does not configure that label for pods, but i think this is correct because they are not directly managed by Helm.

Reply:
Removed app.kubernetes.io/managed-by: {{ .Release.Service }}